spec.html

```diff
@@ -12297,6 +12297,26 @@ <h1>
       1. Return ~unused~.
     </emu-alg>
   </emu-clause>
+
+  <emu-clause id="sec-canbeheldweakly" type="abstract operation">
+    <h1>
+      CanBeHeldWeakly (
+        _v_: an ECMAScript language value,
+      ): a Boolean
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>It returns *true* if and only if _v_ is considered to have an identity suitable for a weak reference. i.e. Is _v_ a valid candidate key of a WeakMap, or entry in a WeakSet, or target of a WeakRef or FinalizationRegistry.</dd>
+    </dl>
+    <emu-alg>
+      1. If _v_ is an Object, return *true*.
+      1. If _v_ is a Symbol and KeyForSymbol(_v_) is *undefined*, return *true*.
+      1. Return *false*.
+    </emu-alg>
+    <emu-note type="editor">
+      <p>Symbols in the GlobalSymbolRegistry (<emu-xref href="#sec-symbol.for"></emu-xref>) are not considered suitable for use as a WeakMap key because they are permanently live. Well-known symbols (<emu-xref href="#sec-well-known-symbols"></emu-xref>) are likely to never be collected, but are nonetheless treated as suitable for weak reference because they are limited to elements of a fixed size list and therefore manageable by a variety of implementation approaches. However, any values associated to a well-known symbol in a live WeakMap is unlikely to be collected and could “leak” memory resources in many implementations.</p>
```

Comment:
"Permanently live" doesn't quite convey the reason. Strictly speaking, since Symbol.for symbols aren't usable as weak targets, they don't participate in the definition of liveness anyway. Now that we've defined identity, I'm inclined to just say it's because they don't have identity.

Reply:
Updated, and also updated the identity section to not say that all symbols have identity.